lib/routes/kemono/index.ts

```diff
@@ -140,13 +140,34 @@ async function handler(ctx) {
                     desc += kemonoFile;
                 }
 
+                let enclosureInfo = {};
+                load(desc)('audio source, video source').each(function () {
```

Comment:
```suggestion
                $('audio source, video source').each(function () {
```
reuse
https://github.com/DIYgod/RSSHub/blob/65865f6eee1a098b2236dd95cfbac36febf065de/lib/routes/kemono/index.ts#L128

Reply:
i tried that at first. it doesn't work, because at the time `$` was last loaded, `desc` did not contain the media elements i'm looking for

i guess i could do another round of `$ = load(desc)`, but im only using it once so it feels unnecessary to assign it